### PR TITLE
Terraform Env Task & Automate for White Lodge

### PR DESCRIPTION
**Pull request by NickGrantX**
# Motivation and Context
We needed to manually terraform White Lodge whenever we made changes. We wanted to automate this process.

# What has changed
When changes to the terraform repo are committed, the CI pipeline will terraform the White Lodge environment automatically, provided the acceptance tests are passing.

# How to test?
Merge it. Cross your fingers.

# Links
Trello: https://trello.com/c/dx8b9cdR

### CODE REVIEW

**Comment by NickGrantX** [comment]
[Build an environment from pipeline (13)](https://trello.com/c/dx8b9cdR/360-build-an-environment-from-pipeline-13)

**Review by LeoKHoward — APPROVED** [review]
🤞🏻

File: pipelines/build-deploy-test-CI-WL.yml [hunk 1]
```diff
@@ -45,6 +45,8 @@ groups:
 
 - name: "Whitelodge Deployments"
   jobs:
+  - "WL Terraform"
+  - "WL Rabbit Helm"
```
Comment (AdamHawtin) [inline pipelines/build-deploy-test-CI-WL.yml:49]: Need to be added to the 'Overview' group as well

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

File: pipelines/build-deploy-test-CI-WL.yml (hunk 1 above)
Comment (NickGrantX) [reply pipelines/build-deploy-test-CI-WL.yml:49]: Done

**Review by AdamHawtin — APPROVED** [review]
👍

**Pull request merged**